assets/js/blocks/featured-items/with-edit-mode.tsx

```diff
@@ -40,9 +73,15 @@ export const withEditMode = ( { description, editLabel, icon, label } ) => (
 				{ description }
 				<div className={ `${ className }__selection` }>
 					{ name === BLOCK_NAMES.featuredCategory && (
+						// Ignoring this TS error for now as it seems that `ProductCategoryControl`
```

Comment:
If we haven't already we could create an issue to revisit `ProductCategoryControl` types. My worry would be that otherwise the ts-ignore might just stick around forever.

Reply:
True. Honestly I hadn't added it because my worry was that instead the ticket would stick around forever, or be lost in the stale oblivion. But you are right.